src/Compilers/VisualBasic/Test/Syntax/Parser/ParserRegressionTests.vb

```diff
@@ -934,4 +934,17 @@ End Enum
         Assert.Equal(Syntax.InternalSyntax.Scanner.BadTokenCountLimit, tree.GetDiagnostics().Where(Function(d) d.Code = ERRID.ERR_IllegalChar).Count())
     End Sub
 
+    <Fact, WorkItem(48587, "https://github.com/dotnet/roslyn/issues/48587")>
+    Public Sub ParseTrailingTextAfterPropertyWithParentheses()
+        ParseAndVerify(<![CDATA[
+Class C
+    Public ReadOnly Property NumberOfResult1() String Integer JohnDoe WwwIIWww Wow
+    Public ReadOnly Property NumberOfResult2() Some unexpected tokens As Integer
+End Class
+]]>,
+<errors>
+    <error id="30205"/>
```

Comment:
><error id="30205"/> [](start = 4, length = 19)

Please switch to form of the test that can use AssertTheseDiagnostics helper (or the like), so that we could see the text of the error and its location